keanu-project/src/main/java/io/improbable/keanu/util/Py4jUtils.java

```diff
@@ -0,0 +1,43 @@
+package io.improbable.keanu.util;
+
+import lombok.experimental.UtilityClass;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+@UtilityClass
+public class Py4jUtils {
```

Comment:
I don't like `Util` very much in the name of classes, they normally become a graveyard for all sorts of stuff. Can we give it a specific name like `Py4jByteConverter`